src/models/applicationState.ts

```diff
@@ -346,6 +347,14 @@ export enum AssetType {
     TIFF = 6,
 }
 
+export enum AssetMimeType {
```

Comment:
There are some libraries that could convert file extensions to content types. Probably we could leverage them. One of the popular one is called mime: https://www.npmjs.com/package/mime

Reply:
wounds good, I'll use that library